nikola/data/themes/base-jinja/templates/gallery.tmpl

```diff
@@ -15,12 +15,26 @@
     </p>
     {% endif %}
     {% if folders %}
-    <ul>
-    {% for folder, ftitle in folders %}
-        <li><a href="{{ folder }}"><i
-        class="icon-folder-open"></i>&nbsp;{{ ftitle|e }}</a></li>
-    {% endfor %}
-    </ul>
+        {% if galleries_use_thumbnail %}
+            {% for (folder, ftitle, fpost) in folders %}
+            <div class="thumnbnail-container">
+                <a href="{{ folder }}" class="thumbnail image-reference" title="{{ ftitle|e }}">\
+                    {% if fpost and fpost.previewimage %}
+                        <img src="{{ fpost.previewimage }}" alt="{{ ftitle|e }}" loading="lazy" style="max-width:{{ thumbnail_size }}px; max-height:{{ thumbnail_size }}px;" />
```

Comment:
This could lead to a path issue. If the user provides an image filename, the previewimage would be valid inside the gallery, but not outside. There is no documentation that states what is required. At the same time, it might be a good idea to include the previewimage as og:image and other open graph metadata in gallery pages.

Reply:
I'll add a mention in the docs about that.

And yeah, it would be nice to have that in og:image, I'll add it in a followup PR

Reply:
Actually, the docs for previewimage metadata say "image path relative to BASE_URL"

Reply:
`open_graph_metadata` uses this to handle all cases:

```mako
<meta property="og:image" content="${url_replacer(permalink, post.previewimage, lang, 'absolute')}">
```

Reply:
Not really sure that would work here since the permalink is not the post's link but the parent folder's?

Reply:
Should work if we pass `folder`, or `permalink+"/"+folder`.

Reply:
Ok, I'll take a shot today during lunch

Reply:
Not doing this because I am not sure how :-(

So, basically: previewimage is meant to be "image path relative to BASE_URL" until further notice.